geonode/upload/utils.py

```diff
@@ -45,11 +45,11 @@ def message(self):
             datastore = settings.OGC_SERVER['default']['DATASTORE']
             pg_geogig_db = settings.DATABASES[datastore]
             message["dbHost"] = pg_geogig_db['HOST']
-            message["dbPort"] = pg_geogig_db['PORT'] or '5432'
+            message["dbPort"] = pg_geogig_db.get('PORT', '5432')
```

Comment:
This 'or' statement wasn't doing anything; get() with the default value is what was intended.